src/interfaces/IChainlinkOracle.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
```

Comment:
We use `>= 0.5.0` in Blue because we want interfaces to be easily usable. No use in breaking in later versions for example